Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -28,6 +28,8 @@
 using BuildXL.Native.IO;
 using BuildXL.Utilities.Collections;
 using BuildXL.Utilities.Tracing;
+using DateTimeUtilities = BuildXL.Cache.ContentStore.Utils.DateTimeUtilities;

```

Comment:
Why? Please no changes to using blocks of random files with no other changes. And in general let's only define namespace aliases if forced to by ambiguity. Is this one of the many bad things that re-sharper does?

Reply:
I moved some functionality from distributed layer into the library into the file with the same type.

I can move the entire DateTImeUtilities into LIbrary though. In this case the only change will be the namespace change.

---
In reply to: [283070517](https://github.com/microsoft/BuildXL/pull/289#discussion_r283070517) [](ancestors = 283070517)